### DISCUSSION TITLE
Roboto Font

### DISCUSSION THREAD

**Discussion by wdebusschere** [mdbootstrap/TW-Elements#1923] [Support from community] [answered]
We just started implementing Tailwind Elements, and this plugin overrides our Poppins font, everywhere is Roboto now, how to make sure we can set the font? Thank you.

**Comment by wdebusschere** [comment]
Ok, think i found it, configure in tailwind.config.js
```
  corePlugins: {
        fontFamily: false
      },
```

Hm, this disables all the fonts

**Comment by juujisai** [comment]
You should be also able to overwrite our font by changing the `body` `font-family` property

```
body {
  font-family: Arial, Helvetica, sans-serif;
}
```

**Comment by wdebusschere** [comment]
@juujisai 
Overriding all will work, but we prefer to do this in the tailwind config file, it's much cleaner, and it's not just body, heading,..
What is the reasing of adding by default Roboto? Is there really no cleaner way to do this?
Thank you.

**Comment by juujisai** [comment, marked as answer]
@wdebusschere We have added the Roboto font as a part of our design.

The other way to change the font is to change it inside `tailwind.config.js` under the `theme` property. For example:
```
  theme: {
    extend: {
      fontFamily: {
        sans: ["Times New Roman", "serif"],
        body: ["Times New Roman", "serif"],
        mono: ["ui-monospace", "monospace"],
      },
    },
  },
```

**Comment by wdebusschere** [comment]
Ok, when i move the fontFamily from the theme directly to  extend it works!
Thank you.
